view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -293,7 +293,7 @@ define(
              * @return {array}
              */
             getInstallmentTerms: function (id) {
-                const installmentBackends = checkoutConfig.installment_backends;
+                const installmentBackends = this.capabilities;
```

Comment:
Out of curiosity, do you take into account that for the [latest version of the Omise capability API ](https://www.omise.co/capability-api?version=2019-05-29) the attribute is `payment_methods` while the older ones are named `payment_backends`

Reply:
lets me help answering this one 
By default our magento will using 2017 api version following this line 
https://github.com/omise/omise-magento/blob/934ae23de1c046aecab55cf68a4685c48b11157d/Model/Omise.php#L58
so that why is will work 🙏